src/js/apps/patients/worklist/filters_app.js

```diff
@@ -62,6 +64,22 @@ export default App.extend({
 
     this.showChildView('owner', ownerFilter);
   },
+  showOwnerToggleView() {
+    if (!this.shouldShowOwnerToggle) return;
+
+    let ownerView;
+
+    if (this.getState('noOwner')) {
+      ownerView = this.showChildView('ownerToggle', new NoOwnerActiveView());
+    } else {
+      ownerView = this.showChildView('ownerToggle', new NoOwnerInactiveView());
+    }
+
+    this.listenTo(ownerView, 'click', () => {
+      this.setState('noOwner', !this.getState('noOwner'));
+      this.showOwnerToggleView();
```

Comment:
I think this could be one view that gets passed `model: this.getState()` and maybe just to simplify things puts the button / classes in the template.  You could use `getTemplate` if it's cleaner than template logic and the view can render for `modelEvents` `change:noOwner`